### DISCUSSION TITLE
Realistic_Multiuser_MIMO_Simulations.ipynb

### DISCUSSION THREAD

**Discussion by alfa-vesta** [NVlabs/sionna#87] [Q&A]
Hi All,

After changing the transmission direction from "uplink" to "downlink" in "Realistic_Multiuser_MIMO_Simulations.ipynb" notebook the BLER/BER become flat with respect to EbNo so any idea why ?

<img width="1177" alt="Screenshot 2022-12-28 at 12 21 18 AM" src="https://user-images.githubusercontent.com/58993631/209718835-93704197-87f3-4576-9408-33aa14d208e8.png">

**Comment by jhoydis** [comment]
Hi,

Can you be more specific with respect to where in the notebook you have made this change?
For downlink transmissions to work, you would probably need to configure some form of transmit precoding, as for example done in this [notebook](https://nvlabs.github.io/sionna/examples/MIMO_OFDM_Transmissions_over_CDL.html#Downlink-Transmission-in-the-Frequency-Domain).

**Reply by alfa-vesta** [reply]
Yes exactly, this what I did. I used ZF precoding for the downlink channel and in "UMi" I set the direction to "downlink"

<img width="1141" alt="Screenshot 2023-01-03 at 7 57 32 PM" src="https://user-images.githubusercontent.com/58993631/210394693-3e7e2984-e307-49bb-9f40-e9f4b2b1ad5a.png">

<img width="1141" alt="Screenshot 2023-01-03 at 7 58 08 PM" src="https://user-images.githubusercontent.com/58993631/210394757-a494f399-42fa-4de1-aa2b-acb152d4995a.png">

**Reply by jhoydis** [reply]
It is very difficult to assess the source of the behaviour from this code snippet. Could you share the full notebook?

**Reply by alfa-vesta** [reply]
pls find attached notebook
[DL_MIMO_Simulation.ipynb.zip](https://github.com/NVlabs/sionna/files/10347088/DL_MIMO_Simulation.ipynb.zip)
